### DISCUSSION TITLE
Is Quasar v1 compatible with Vite and Vue2?

### DISCUSSION THREAD

**Discussion by dreams-and-thoughts** [quasarframework/quasar#15564] [General - Components / Directives / etc] [answered]
Looking into migrating an existing project that uses Quasar v1 and Vue2 CLI across to Vite, instead of Webpack based Vue-CLI.

Partway through the migration, and I'm getting some errors related to Quasar:

```
Error: Cannot find module 'quasar/dist/transforms/auto-import.json'
Require stack:
- ./node_modules/@quasar/vite-plugin/dist/index.js
```

Critically, perhaps, the project is also Typescript :(

Is Quasar v1 compatible with Vite?

If not, and I have to upgrade Quasar, does that also mean I'd have to upgrade from Vue2 to Vue3?

I used this reference to setup vite-plugin: (Which I can now see is for v2, so can someone please just clarify this for me and perhaps suggest best way forward? )

https://quasar.dev/start/vite-plugin

**Comment by rstoenescu** [comment, marked as answer]
Quasar v1 is not compatible with Vite. However you can follow the upgrade instructions and have Vite working with v2.

**Reply by dreams-and-thoughts** [reply]
Thanks. How about Vue2, will I need to upgrade that as well?

**Reply by MilosPaunovic** [reply]
Yes, Quasar 2 uses Vue 3.

**Reply by rstoenescu** [reply]
https://quasar.dev/start/upgrade-guide#migrate-to-v2-from-v1
